venom/rpc/reflect/openapi.py

```diff
@@ -1,6 +1,195 @@
+from itertools import groupby
+from functools import singledispatch
+from collections import defaultdict
 from venom.rpc.reflect.reflect import Reflect
-from venom.rpc.reflect.stubs import OpenAPISchema
+from venom.rpc.reflect.stubs import TypeMessage, MethodMessage, \
+    ParameterMessage, ResponsesMessage, ResponseMessage, \
+    FieldsMessage, InfoMessage, OpenAPISchema
+from venom.protocol import JSON
+from venom.fields import Field, RepeatField, MapField
+from venom import Message
+from venom.message import field_names, fields
+from venom.rpc.method import Method, HTTPFieldLocation
+
+
+DEFINITIONS = 'definitions'
```

Comment:
These won't change and they are only used in one or two places. It's more legible to use the strings instead.